apps/11gist-blog/script.js

```diff
@@ -1,5 +1,30 @@
 'use strict';
 
 $(document).ready(function() {
-  // You code here
+  $.ajax('http://127.0.0.1:8080/apps/11gist-blog/api/gists.json', {
+    success: function(userPosts) {
+      userPosts.forEach(function(userGist) {
+        var userSubstr = userGist.description; 
+        var postString = "<li>" + 
```

Comment:
Only 2 posts should pass through; the 2 that start with `"#post "`. Use an `if` statement to filter your posts.